app/models/computacenter/responsible_body_urns.rb

```diff
@@ -48,6 +48,9 @@ def computacenter_identifier
         else
           "t#{companies_house_number.to_i}"
         end
+      when 'FurtherEducationCollege'
+        # potential n+1
+        schools.first&.ukprn
```

Comment:
Is it worth looking at whether we're eager loading schools where we use this method?

Reply:
I had a look for uses of `computacenter_identifier` and most seem to be referring to one RB or if it is a list it's a short list. The only exception is the "closed schools" in the `/computacenter` area. Given this and there are not many FE bodies for the moment I think this is safe for now

Reply:
the "closed schools" area has been removed now so don't think this should cause an issue at least for now